lua/nvim_lsp/angular.lua

```diff
@@ -0,0 +1,72 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+local server_name = 'angularls'
```

Comment:
File name should be the same with the server name.
Would you rename `angular.lua` to `angularls.lua`?